cfme/tests/infrastructure/test_infra_manual.py

```diff
@@ -82,3 +82,249 @@ def test_provider_summary_topology():
         1532404
     """
     pass
+
+
+@pytest.mark.manual
+@test_requirements.infra_hosts
+def test_setting_default_filter():
```

Comment:
I would consider changing this to `test_host_setting_default_filter` to match the other test cases you've defined here